lib/content-services/src/lib/node-comments/services/node-comments.service.ts

```diff
@@ -73,15 +75,25 @@ export class NodeCommentsService implements CommentsService {
     }
 
     private newCommentModel(comment: Comment): CommentModel {
+        const user = new User(comment.createdBy);
+        const userId = user.id?.toString();
+
+        const imageId = user.pictureId?.toString() || user.avatarId;
```

Comment:
Why are we checking `pictureId` here? As far as I can see this is the activiti user property